org.jousse.vincent.Pomodorolm.yml

```diff
@@ -0,0 +1,45 @@
+id: org.jousse.vincent.Pomodorolm
+
+runtime: org.gnome.Platform
+runtime-version: "46"
+sdk: org.gnome.Sdk
+
+command: pomodorolm
+finish-args:
+  - --socket=wayland # Permission needed to show the window
+  - --socket=fallback-x11 # Permission needed to show the window
+  - --socket=pulseaudio # We want to be able to play sounds
+  - --device=dri # OpenGL
+  - --share=ipc
+  - --env=APPDIR=/app # Fake AppImage env for Tauri. Needed so than it will look for assets in /app
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  # Required for the systray to work
+  - --own-name=org.kde.*
+  # Needed for the systray to work properly. libappindicator emits a dbus signal with the systray icon absolute path
+  # so flatpak and the host need to both have access to the icon for the systray to work properly
+  - --filesystem=xdg-data
```

Comment:
this won't do, you should not ask for access to entire xdg-data. At best the icon could be in a subdirectory or in tmp folder.

Reply:
Another approach is that you just supply the icon name for tray: `org.jousse.vincent.Pomodorlm-tray.png`, appindicators can pick them up as Flatpak will export them and set `XDG_
DATA_DIRS`

Reply:
Well the problem is that I'm using [Tauri](https://v2.tauri.app) and I don't have access directly to the appindicator. Sharing `xdg-data/tray-icon` between the two should do the trick I think.